_backend/composer.json

```diff
@@ -7,7 +7,7 @@
     },
     "require": {
         "stripe/stripe-php": "7.108.0",
-        "mandrill/mandrill": "^1.0",
+	"mailchimp/transactional": "*",
```

Comment:
Can we pin this version to the latest stable release, instead of using a wildcard?